.github/workflows/preview-web.yaml

```
@@ -26,6 +26,14 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
+  Analyze bundle sizes:
```

actionlint (1.7.7)

invalid job ID "Analyze bundle sizes". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)